metadata.json

```diff
@@ -2,87 +2,87 @@
  {
   "contractName": "BasicFDT",
   "contractSize": 0,
-  "sourceHash": "0x67502ea16139de7c6a7d24101169d931973253f76a2a85bbc4a63da4fa8e6123",
+  "sourceHash": "0xd718410027ac8b1e9f557334d8236cca88ebe89c028eaddced0c42d03e608be4",
```

Comment:
Is it essential to push this file?

Reply:
No, it is not. Ideall CI will build the old and new and allow them to be compared as a requirement for merging a PR.